.travis.yml

```diff
@@ -1,26 +1,28 @@
 language: node_js
 node_js: '10'
 cache:
+  yarn: true
   directories:
-    - ~/.npm
+    - node_modules
+    - ~/.cache
 env:
   global:
     - FORCE_COLOR=1
 install:
-  - npm ci
+  - yarn
```

Comment:
Doesn't yarn have an equivalent for `npm ci` (strict lockfile)?

Reply:
I added a `.yarnrc` with `--install.frozen-lockfile true`. More info:

https://yarnpkg.com/en/docs/yarnrc#toc-cli-arguments
https://github.com/yarnpkg/yarn/issues/4147#issuecomment-420769373
https://yarnpkg.com/lang/en/docs/cli/install/#toc-yarn-install-frozen-lockfile

Reply:
> On yarn install the default behavior is that the yarn.lock file is mutated if package.json and yarn.lock are out of sync:

This sounds desirable for local development. I would want yarn/npm to fail in CI if they are out of sync, but reconcile them locally, so I would propose we only use the CLI flag in CI instead of a yarnrc

Reply:
Oh right! Fixed.